src/airtable/sync_airtable_all_proposal_votes_snapshot.js

```diff
@@ -75,21 +66,23 @@ const getAllProposalVotes = async () => {
 
 // This updates every record with the latest snapshot votes
 const main = async () => {
-  await getAllProposalVotes()
-  Logger.log('\n============ Total Proposal Scores [%s]', proposalScores.length)
+  return console.error("Don't run me")
+
+  // await getAllProposalVotes()
+  // Logger.log('\n============ Total Proposal Scores [%s]', proposalScores.length)
 
-  proposalVoteSummary = await sumSnapshotVotesToAirtable(
-    allProposals,
-    proposalScores
-  )
-  Logger.log('\n============ Total Proposals [%s]', proposalVoteSummary.length)
+  // proposalVoteSummary = await sumSnapshotVotesToAirtable(
+  //   allProposals,
+  //   proposalScores
+  // )
+  // Logger.log('\n============ Total Proposals [%s]', proposalVoteSummary.length)
 
-  await updateProposalRecords(proposalVoteSummary)
-  Logger.log(
-    '[%s]\nUpdated [%s] rows to Airtable',
-    new Date().toString(),
-    proposalVoteSummary.length
-  )
+  // await updateProposalRecords(proposalVoteSummary)
+  // Logger.log(
+  //   '[%s]\nUpdated [%s] rows to Airtable',
+  //   new Date().toString(),
+  //   proposalVoteSummary.length
+  // )
 }
 
 main()
```

Comment:
Ok, I agree. I don't believe this is being used anymore. I'll try to remove functionality as soon as it's deprecated. We can always do github code archeology.